ldk/goja/src/ui/index.ts

```diff
@@ -0,0 +1,14 @@
+export interface UI {
+  listenSearchbar(cb: (val: string) => void): void;
+  listenGlobalSearch(cb: (val: string) => void): void;
```

Comment:
I think it will be valuable to add comments for the public interfaces.

Reply:
We are adding them as follows:

`/**
     * Returns true if the specified vault entry exists.
     * 
     * @param key - The key for the item to check existence.
     * @returns True if the specified vault entry exists.
     */
    exists(key: string): Promise<boolean>;`